presto-main/src/main/java/io/prestosql/testing/LocalQueryRunner.java

```diff
@@ -363,7 +364,8 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
                 new EventListenerManager(),
                 blockEncodingManager,
                 new SessionPropertyDefaults(nodeInfo),
-                typeRegistry);
+                typeRegistry,
+                new StatementAccessControlManager());
```

Comment:
could yo put `StatementAccessControlManager` below `EventListenerManager` in `PluginManager` arguments?

Reply:
Done